source/releasenotes/2024-12-10-composer-2-8-3.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Composer 2.8.3 now available
+published_date: "2024-12-10"
+categories: [infrastructure,tools-apis]
+---
+
+Composer 2.8.3 is now available on Pantheon as of December 10, 2024.
+
+### No action required
+
+Composer 2.8.3 will slowly roll out to all Integrated Composer sites on Pantheon automatically. There is no action required on your part.
```

Comment:
I think Composer is globally installed on all sites, not just those with IC.

Reply:
Yes but I don't think you would see or interact with it if you weren't using IC.

Reply:
In fact we should clarify if this change is to
- runtimes
- Integrated Composer build containers
- Both?